aqueduct_client/errors.py

```diff
@@ -0,0 +1,29 @@
+class ConcurrentExecutionsExceeded(Exception):
+    def __init__(self, concurrent_executions_count, allowed_executions_count):
+        self._current = concurrent_executions_count
```

Comment:
I would probably just make these public fields called `current` and `allowed` (and document them in an `Attributes` section of the class docstring).

Reply:
will do